tests/e2e/e2e_main.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is an end-to-end test. Test will ssh to a vm and create a volume using docker cli.
+// After creating a volume, test will ssh to the ESX and verify the properties of the volume.
+// Properties being verified - capacity, disk-format and vm-attached field.
+
+// Test assumes that SSH cert has been setup to enable password-less login to VM and ESX.
+
+package e2e
+
+import (
+	"testing"
+	"os"
+)
+
+var DockerHosts = []string{os.Getenv("VM2"), os.Getenv("VM1")}
+var Host, Vmgroup, Ds1, Ds2, Cname string
+
+func TestMain(m *testing.M) {
```

Comment:
Is this TestMain introduced for vmgroup_volumeops_test.go only? If the answer is yes, then shouldn't we move these code into vmgroup_volumeops_test.go, or maybe just give this file a better (more specific) name? The name "e2e_main" is too generic which should be used for the whole e2e tests only.